src/Auth/LoginContainer.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import Box from 'grommet/components/Box';
+import Button from 'grommet/components/Button';
+import Footer from 'grommet/components/Footer';
+import Form from 'grommet/components/Form';
+import FormField from 'grommet/components/FormField';
+import Heading from 'grommet/components/Heading';
+import TextInput from 'grommet/components/TextInput';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { changeUsername, loginRequest } from './authModule';
+
+class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.onChangeUsername = this.onChangeUsername.bind(this);
+    this.onFormLogin = this.onFormLogin.bind(this);
+  }
+
+  onChangeUsername(event) {
+    const username = event.target.value.trim();
+    this.props.changeUsername({ username });
+  }
+
+  onFormLogin() {
+    const { username } = this.props;
+    if (username.length === 0) return;
+    this.props.loginRequest();
+  }
+
+  render() {
+    return (
+      <Box align="center">
+        <Form pad="medium" fill>
+          <Heading align="center" strong>Kizuna</Heading>
+          <Button
+            label="Facebook"
+            fill
+            onClick={() => {}}
```

Comment:
> Because I have not done it yet, I just put the button for the true position.

From this, you should get a feedback that the scope of your issue is too big.